src/jio-navbar.ts

```diff
@@ -197,6 +197,9 @@ export class Navbar extends LitElement {
     let searchboxHtml = null;
     if (this.showSearchBox) {
       searchboxHtml = html`<jio-searchbox></jio-searchbox>`;
+      if (this.menuToggled) {
+        this._clickCollapseButton;
```

Comment:
I'm not sure what the intention here is.

1) _clickCollapseButton is a function, so it should be this._clickCollapseButton()
2) This is the render function, so every time the component renders at all, it'll toggle the menu 
3) showSearchBox is a attribute to decide if the searchbox should be shown on the menu bar, its not an action as you sorta think it is.

You may be able to get at what you are trying to do by adding an onclick to jio-searchbox, or somehow add an event handler to the algolia stuff.

Reply:
I had misunderstood certain aspects. I shall add onclick to jio-searchbox as you have advised. Thanks for the explanation.